modules/distr-tracing-tempo-config-default.adoc

```diff
@@ -4,147 +4,58 @@
 
 :_mod-docs-content-type: REFERENCE
 [id="distr-tracing-tempo-config-default_{context}"]
-= Default configuration options
+= Introduction to TempoStack configuration parameters
 
 The `TempoStack` custom resource (CR) defines the architecture and settings to be used when creating the {TempoShortName} resources. You can modify these parameters to customize your {TempoShortName} implementation to your business needs.
 
-.Example of a generic Tempo YAML file
+.Example TempoStack CR
```

Comment:
Another one that's more of a comment for the future - in the following example, I would provide actual examples of the configuration for each field, rather than `{}` for each, since this doesn't really provide a lot of information for users.

Reply:
Thank you. Noted.